hack/code/bandwidth_gen/main.go

```diff
@@ -32,12 +32,13 @@ import (
 )
 
 var uriSelectors = map[string]string{
-	"https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/general-purpose-instances.html":            "#general-purpose-network-performance",
-	"https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/compute-optimized-instances.html":          "#compute-network-performance",
-	"https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/memory-optimized-instances.html":           "#memory-network-perf",
-	"https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/storage-optimized-instances.html":          "#storage-network-performance",
-	"https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/accelerated-computing-instances.html":      "#gpu-network-performance",
-	"https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/high-performance-computing-instances.html": "#hpc-network-performance",
+	"https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/general-purpose-instances.html":            "#gp_network",
+	"https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/compute-optimized-instances.html":          "#co_network",
```

Comment:
This redirects me to https://docs.aws.amazon.com/ec2/latest/instancetypes/co.html, can we update each of these to match where they're redirecting too? 
```suggestion
	" https://docs.aws.amazon.com/ec2/latest/instancetypes/co.html":          "#co_network",
```